examples/common/boilerplate.rs

```diff
@@ -164,7 +164,9 @@ pub fn main_wrapper<E: Example>(
         DeviceUintSize::new(size.width as u32, size.height as u32)
     };
     let notifier = Box::new(Notifier::new(events_loop.create_proxy()));
-    let (mut renderer, sender) = webrender::Renderer::new(gl.clone(), notifier, opts).unwrap();
+    let upload_method = webrender::UploadMethod::PixelBuffer(webrender::VertexUsageHint::Stream);
```

Comment:
Let's make the ```upload_method``` argument to ```Device::new``` be ```Option<UploadMethod>``` and then inside the constructor, we could have something like:
```
let upload_method = upload_method.unwrap_or( ... pixel buffer / stream default ... );
```

That way clients don't need to specify an upload method unless they care.